roles/provision/defaults/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
-postgresql_rpm_gpg_key_pgdg: "https://www.postgresql.org/download/keys/RPM-GPG-KEY-PGDG" # not needed for CentOS 7
+postgresql_rpm_gpg_key_pgdg: "https://apt.postgresql.org/pub/repos/yum/keys/PGDG-RPM-GPG-KEY-RHEL" # not needed for CentOS 7
```

Comment:
I think keeping it as a single variable (rather than nested) makes sense, and you can then override the default in your inventory group vars to specify the ARM key

Reply:
But what about the (my) use case where locally I want ARM but production I don't? I don't think what you suggest is a particularly satisfying solution.

Reply:
Oh I see, I didn't realise that's what you were after. Yeah we should probably support both architectures then. But ideally the variable names would be flat rather than nested (to make it easier to change one without needing to change both):
```yaml
postgresql_rpm_gpg_key_pgdg_x86_64: https://apt.postgresql.org/pub/repos/yum/keys/PGDG-RPM-GPG-KEY-RHEL
postgresql_rpm_gpg_key_pgdg_aarch64: https://apt.postgresql.org/pub/repos/yum/keys/PGDG-RPM-GPG-KEY-AARCH64-RHEL
```

Reply:
Good idea